### PR TITLE
Update dependencies & plugins

### PR DESCRIPTION
**Pull request by wendigo**
Compilation tested against both Trino and Airlift

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ Airbase 148
 * Dependency updates:
   - Jackson 2.15.3 (from 2.15.2)
 
+* Plugin updates:
+  - errorprone 2.22.0 (from 2.21.1)
```
Comment (electrum) [inline CHANGES.md:13]: This is actually a dependency update. We don't (yet) have an Error Prone plugin configuration in Airbase.

**Review by electrum — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (wendigo) [reply CHANGES.md:13]: I'll correct changelog

**Comment by wendigo** [comment]
@electrum fixed, I've also added new jacoco plugin which supports JDK 21

**Comment by wendigo** [comment]
I've added 3 more plugin updates

**Pull request merged**